config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```
@@ -1,7 +1,8 @@
 periodics:
-  - name: periodic-kubernetes-unit-test-ppc64le
+  - name: periodic-kubernetes-unit-test-nonroot-ppc64le
```

no need mention nonroot in the name

```suggestion
  - name: periodic-kubernetes-unit-test-ppc64le
```